frontend/src/contexts/toastAuthorizationResult.tsx

```diff
@@ -0,0 +1,37 @@
+import { ToastTypes } from "@/enums/ToastType";
+import React from "react";
+import { toast } from "sonner";
+
+export const toastAuthorizationResult = (
+  toastType: ToastTypes,
+  toastTitle: string,
```

Comment:
Just a thought—why not make this field optional? If `toastType` and `toastTitle` are often the same, we could allow `toastType` to serve as the `toastTitle` when `toastTitle` isn’t defined.

Reply:
Done